server/resolvers/RequiredReportOperations/updateRequiredReportResolver.js

```diff
@@ -0,0 +1,37 @@
+const { AuthenticationError } = require('apollo-server');
+const { updateAtBrowser } = require('../../models/services/AtBrowserService');
+
+const updateRequiredReportResolver = async (
+    { parentContext: { atId, browserId, phase } },
+    { atId: inputAtId, browserId: inputBrowserId },
+    { user }
+) => {
+    if (!user?.roles.find(role => role.name === 'ADMIN')) {
+        throw new AuthenticationError();
+    }
+
+    let updateParams = {};
+
+    if (phase === 'IS_CANDIDATE') {
+        updateParams = { isCandidate: false };
```

Comment:
Modifying `updateParams` and calling `updateAtBrowser` twice for the parent context and input values strikes me as very odd. Since it isn't self-explanatory, could you add a comment explaining why this decision was made? This will help future contributors (and present me) make sense of it.

Reply:
Comment added!